src/tracing/CMakeLists.txt

```diff
@@ -43,9 +43,9 @@ endfunction()
 
 set(osd_traces oprequest.tp osd.tp pg.tp)
 add_tracing_library(osd_tp "${osd_traces}" 1.0.0)
-add_tracing_library(rados_tp librados.tp 2.0.0)
+add_tracing_library(librados_tp librados.tp 2.0.0)
```

Comment:
Yes, I see, thanks! Now, I have no idea how to handle this... how about revoking commit: 0a71759?